tests/test_discovery.py

```diff
@@ -223,16 +223,73 @@ def test_subactors_unregister_on_cancel_remote_daemon(
             spawn_and_check_registry,
             arb_addr,
             use_signal,
-            True,
+            True,  # remote arbiter
```

Comment:
Opaque booleans make for great required keyword arguments.

Reply:
@ryanhiebert i think the only reason for this was avoiding a `funtools.partial`.

Maybe it's better to be clearer and not worry about the extra import tho?

Reply:
I wouldn't worry about an extra import. Clarity beats that for sure. It's quite possible there's other worthwhile considerations, but an import seems like a concern to not have.

Reply:
lol nm it was already imported so you win :stuck_out_tongue_closed_eyes: